pkg/operator/operator.go

```diff
@@ -480,8 +482,12 @@ func (o *Operator) sync(key string) error {
 		return err
 	}
 
+	var degradedConditionMessage string
+	if !config.IsStorageConfigured() {
+		degradedConditionMessage = StorageNotConfiguredMessage
+	}
 	klog.Info("Updating ClusterOperator status to done.")
```

Comment:
nit: Can you update this log to reflect the status instead of done?

Reply:
The status will be done, it's just that we will add another message to the degraded condition. We can output an log to indicate this, but I would not remove this line.